app/controllers/media-controller.js

```diff
@@ -9,6 +9,10 @@ const request = require( 'request' );
 const express = require( 'express' );
 const router = express.Router();
 const debug = require( 'debug' )( 'media-controller' );
+const { RequestFilteringHttpAgent } = require( 'request-filtering-agent' );
+
+const config = require('../../config/config');
```

Comment:
Use `require('../models/config-model').server`. Configs can also be created with environment variables, and there is some cleaning/manipulation done in the config-model.